src/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from './components/NavigationTree';
 export * from './components/TreeView';
+// TODO: remove in the next major version
 export {i18n} from './components/i18n';
```

Comment:
this export was incorrect, should've provided the configure function right away
but deleting it would break public API, and there is no harm in exporting it, so lets keep it until the next major I guess